meiki_server/auth/routes_test.go

```diff
@@ -113,7 +113,7 @@ func (s *AuthRoutesTestSuite) TestRoutesScenario() {
 		Password: "no-shnoo",
 	})
 	req, _ = http.NewRequest("POST", "/login", bytes.NewBuffer(badCredentialsBody2))
-	s.assertStatusCode(req, 401)
+	s.assertStatusCode(req, 400)
```

Comment:
Why is this 400 instead of 401?

Reply:
400 because username does not exist and not password is wrong? (not sure on this one tho)

Reply:
Hey, I do not know what PR this is, presumably this got killed when I did some git-fu, the actual PR is here: https://github.com/RainingComputers/Meiki/pull/2